force-app/main/default/flexipages/Data_Service.flexipage-meta.xml

```diff
@@ -11,6 +11,9 @@
         <componentInstances>
             <componentName>ldsCreateRecord</componentName>
         </componentInstances>
+        <componentInstances>
+            <componentName>generateRecordInputForCreate</componentName>
```

Comment:
I've changed it. However, now, on one hand, it would make more sense to move it to the `wire` tab. But, on another hand, this shows how to use createRecord but populating the defaults calling `generateRecordInputForCreate`, so it also makes sense to have it where it is. What about something like.... `ldsCreateRecordWireDefaults`?

Reply:
Considering changing the name to "wireGetRecordCreateDefaults" as it's really the function that we wire

Reply:
Finally changed to "ldsGenerateRecordInputForCreate" as agreed